deps/verifier/src/tdx/claims.rs

```diff
@@ -239,8 +240,20 @@ fn parse_ccel(ccel: CcEventLog, ccel_map: &mut Map<String, Value>) -> Result<()>
     Ok(())
 }
 
-const ERR_INVALID_HEADER: &str = "invalid header";
-const ERR_NOT_ENOUGH_DATA: &str = "not enough data after header";
+#[derive(Error, Debug, PartialEq)]
+pub enum ParseError {
+    #[error("Failed to parse `Descriptor`")]
+    ParseDescriptor,
+
+    #[error("Failed to parse `InfoLength`")]
+    ParseInfoLength,
```

Comment:
AFAIU, it's more like:
```suggestion
    ReadInfoLength(#[from] io::Error),
```

Reply:
`std::io:Error` does not support `PartialEq` thus we cannot use `#[from]` macro. This is possible as the error information is enough.